airbyte-integrations/connectors/source-braintree-singer/sample_files/configured_catalog.json

```diff
@@ -193,8 +193,12 @@
             }
           }
         },
-        "supported_sync_modes": ["full_refresh"]
-      }
+        "source_defined_cursor": true,
+        "supported_sync_modes": ["incremental"],
+        "default_cursor_field": []
+      },
+      "sync_mode": "incremental",
+      "cursor_field": []
```

Comment:
Why "cursor_field" and "default_cursor_field" are empty?

Reply:
my guess is that the cursor field is not an actual field in the returned stream?

Reply:
Yes, you are right, I have already correct this mistake